### PR TITLE
add view_payments org feature, allow any org member to see /settings

### PR DESCRIPTION
**Pull request by camhashemi**
@ryanpbrewster @andrewconner @leogrim

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/model/OrganizationSettings.scala [hunk 1]
```diff
@@ -164,4 +165,10 @@ object Feature {
     val permission = OrganizationPermission.VIEW_SETTINGS
     val settings: Set[FeatureSetting] = Set(ADMINS, MEMBERS)
   }
+
+  case object ViewPayments extends Feature with FeatureWithPermissions {
+    val value = OrganizationPermission.VIEW_PAYMENTS.value
+    val permission = OrganizationPermission.VIEW_PAYMENTS
+    val settings: Set[FeatureSetting] = Set(DISABLED, ADMINS)
```
Comment (camhashemi) [inline kifi-backend/modules/common/app/com/keepit/model/OrganizationSettings.scala:172]: `view_payments` is an un-editable feature set to `disabled` on free plans and `admins` on paid plans. it allows admins to view and make actions regarding payments.

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html [hunk 2]
```diff
@@ -105,7 +105,7 @@
             ui-sref="orgProfile.settings.team"
             ng-click="onClickTrack($event, 'clickedProfileSettings')"
             ng-class="{ 'active': state.current.activetab ==='settings'}"
-            ng-if="viewer.permissions.indexOf(ORG_PERMISSION.VIEW_SETTINGS) !== -1"
+            ng-if="viewer.membership"
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html:108]: Why? Isn't this exactly what we want to check?

Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html:108]: we're creating a new settings sub-tab called "earn credit" that's viewable to all members and `view_settings` is becoming a configurable feature, potentially blocked to members. so members need to be able to see the tab without having `view_settings`.

Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html:108]: in product, what used to be the subtab called "member settings" is now "member privileges" to clear up the vocabulary, we could do that on the back-end as well.

File: kifi-backend/modules/common/app/com/keepit/model/OrganizationSettings.scala (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/OrganizationSettings.scala:172]: Why? How would someone upgrade plans? Not sure why this is a setting.

Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/OrganizationSettings.scala:172]: All billing contacts / admins should be able to see it, right?

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html (hunk 2 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html:108]: Might be missing it, but why do we need `view_settings` then?

File: kifi-backend/modules/common/app/com/keepit/model/OrganizationSettings.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/common/app/com/keepit/model/OrganizationSettings.scala:172]: `manage_plan` is the permission we give to admins regardless of plan, `view_payments` is the permission we give to admins when they are on a paid plan.

https://app.asana.com/0/32728208409723/54671767614550 specifies what admins can do, given a plan. `manage_plan` will allow them to upgrade their plan, but they can't see billing contacts unless they're on a paid plan and have the `view_payments` permission.

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html (hunk 2 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html:108]: `view_settings` would be used to enforce whether a user can see https://www.kifi.com/kifi/settings#team-profile-, which is now a toggle-able setting

File: kifi-backend/modules/common/app/com/keepit/model/OrganizationSettings.scala (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/OrganizationSettings.scala:172]: Why separate these? Either someone's in charge / trusted to deal with billing things, or not. What the use case? Someone changing plans but not being able to see billing contacts? Or someone seeing billing contacts and not able to change plans?

If `view_payments` is only about viewing billing contacts, the naming is misleading.

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html (hunk 2 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html:108]: In this case, "settings" (in `view_settings`) does not mean anything with viewing (it means being able to change the name, description, etc, right?) or settings. Did I miss something?

**Comment by camhashemi** [comment]
sync'd with Jen and @andrewconner, `view_payments` doesn't make sense and what it was intended for isn't necessary anymore. Will open another PR that refactors `view_settings` -> `view_permissions` and makes it an editable feature.

**Pull request closed without merging**